apps/api/docker/Dockerfile

```diff
@@ -9,15 +9,15 @@ FROM installer AS runner
 COPY .gitignore .gitignore
 COPY --from=installer /app/out/full .
 
+WORKDIR /app/apps/api
+
 RUN --mount=type=cache,id=pnpm,target=/pnpm/store \
     if [ "$NODE_ENV" = "production" ]; then \
         pnpm install --prod --frozen-lockfile; \
     else \
         pnpm install; \
     fi
 
-WORKDIR /app/apps/api
```

Comment:
why this change?

Reply:
It wasn't installing the correct required packages, resulting in execution errors. Installing them after changing the workdir fixed it.